index.js

```diff
@@ -183,13 +183,26 @@ var jade_encode_html_rules = {
   '"': '&quot;'
 };
 var jade_match_html = /[&<>"]/g;
+/* istanbul ignore next */
 function jade_encode_char(c) {
   return jade_encode_html_rules[c] || c;
 }
 exports.escape = jade_escape;
-function jade_escape(html){
-  var result = String(html).replace(jade_match_html, jade_encode_char);
-  if (result === '' + html) return html;
+function jade_escape(_html){
+  var html = String(_html);
```

Comment:
Style nitpick - on the other PR we used `'' + val`, here we use `String(val)`. Could we settle for one way?